### PR TITLE
start of Profile showing

### PR DESCRIPTION
**Pull request by Saryx11**

### CODE REVIEW

File: DocContact/DocContact/DetailViewController.swift [hunk 1]
```diff
@@ -24,8 +24,18 @@ class DetailViewController: UIViewController {
     weak var contact: Contact?
     weak var delegate: DetailViewControllerDelegate?
     
+    //variables pour afficher le profil de l'utilisateur
+    var user: User?
+    let DBManager = ManageDbProvider.sharedInstance
+    var isContactsDetails: Bool = false
```
Comment (Ailelame) [inline DocContact/DocContact/DetailViewController.swift:30]: not clear

File: DocContact/DocContact/Providers/manageDBProvider.swift [hunk 2]
```diff
@@ -37,6 +37,17 @@ class ManageDbProvider{
         }
         print("L'utilisateur a été ajouté à la base")
     }
+    func getUser()->User?{
+        let fetchRequestUser = NSFetchRequest<User>(entityName : "User")
+        let managedContext = (UIApplication.shared.delegate as! AppDelegate).persistentContainer.viewContext
+        do {
+            let fetchedUser = try managedContext.fetch(fetchRequestUser)
+            print(fetchedUser.first?.lastName)
```
Comment (Ailelame) [inline DocContact/DocContact/Providers/manageDBProvider.swift:45]: delete the print ?

File: DocContact/DocContact/DetailViewController.swift (hunk 1 above)
Comment (Saryx11) [reply DocContact/DocContact/DetailViewController.swift:30]: Not used for the moment, it's just a base for the next step

File: DocContact/DocContact/Providers/manageDBProvider.swift (hunk 2 above)
Comment (Saryx11) [reply DocContact/DocContact/Providers/manageDBProvider.swift:45]: We can see for code cleaning a little later

**Pull request merged**